opensciencegrid/osdf-ospool-issuer/Dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM hub.opensciencegrid.org/opensciencegrid/software-base:3.6-el8-release
+
+RUN yum install -y curl java-11-openjdk java-11-openjdk-devel
+
+# Download and install tomcat
+RUN useradd -r -s /sbin/nologin tomcat ;\
+mkdir -p /opt/tomcat ;\
+curl -s -L https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.69/bin/apache-tomcat-9.0.69.tar.gz | tar -zxf - -C /opt/tomcat --strip-components=1 ;\
+chgrp -R tomcat /opt/tomcat/conf ;\
+chmod g+rwx /opt/tomcat/conf ;\
+chmod g+r /opt/tomcat/conf/* ;\
+chown -R tomcat /opt/tomcat/logs/ /opt/tomcat/temp/ /opt/tomcat/webapps/ /opt/tomcat/work/ ;\
+chgrp -R tomcat /opt/tomcat/bin /opt/tomcat/lib ;\
+chmod g+rwx /opt/tomcat/bin ;\
+chmod g+r /opt/tomcat/bin/*
+
+ADD server.xml /opt/tomcat/conf/server.xml
+RUN chgrp -R tomcat /opt/tomcat/conf/server.xml ;\
+chmod go+r /opt/tomcat/conf/server.xml
+
+ARG TOMCAT_ADMIN_USERNAME=admin
+ARG TOMCAT_ADMIN_PASSWORD=password
+ADD tomcat-users.xml.tmpl /opt/tomcat/conf/tomcat-users.xml.tmpl
+RUN sed s+TOMCAT_ADMIN_USERNAME+${TOMCAT_ADMIN_USERNAME}+g /opt/tomcat/conf/tomcat-users.xml.tmpl | sed s+TOMCAT_ADMIN_PASSWORD+${TOMCAT_ADMIN_PASSWORD}+g > /opt/tomcat/conf/tomcat-users.xml ;\
+chgrp tomcat /opt/tomcat/conf/tomcat-users.xml
+
+ARG TOMCAT_ADMIN_IP=127.0.0.1
```

Comment:
This hurts my eyes. Can you please add a comment to the file explaining why baking the admin credentials into the image like this is OK?